packages/chart/src/Column.ts

```diff
@@ -217,11 +232,17 @@ export interface Column {
     useClonedPalette(_: boolean): this;
     showValue(): boolean;
     showValue(_: boolean): this;
+    centeredValues(): boolean;
+    centeredValues(_: boolean): this;
+    hideCenteredValueByHeight(): number;
+    hideCenteredValueByHeight(_: number): this;
 }
 
 Column.prototype.publish("paletteID", "default", "set", "Palette ID", () => Column.prototype._palette.switch(), { tags: ["Basic", "Shared"] });
 Column.prototype.publish("useClonedPalette", false, "boolean", "Enable or disable using a cloned palette", null, { tags: ["Intermediate", "Shared"] });
 Column.prototype.publish("showValue", false, "boolean", "Show Value in column");
+Column.prototype.publish("centeredValues", false, "boolean", "Show Value in center of column");
+Column.prototype.publish("hideCenteredValueByHeight", null, "number", "Hide centered value if the rect height is less than this value");
```

Comment:
Remove:
* If `valueCentered` is `false` and there is not enough room to display the text at the top / right of the column / bar, then it should be displayed inside the column / bar
* If `valueCentered` is `true` and there is not enough room to display the text inside the column / bar, then it should be displayed as if `valueCentered` === `false` (IOW at the top / right of the column / bar)

Reply:
This could cause confusion when bars are stacked.

Reply:
...and when there are several stacked bar/column rectangles with a small height the labels will overlap with that approach. This implementation works as expected and provides the more flexibility for this situation to be controlled at the application layer.

Reply:
Yes it might:
* If stacked AND valueCentered === true AND value doesn't fit, then hide it.
* If stacked AND valueCentered === false then the chart will be confusing (IMO)